redbot/cogs/trivia/data/lists/worldcup.yaml

```diff
@@ -12,215 +12,221 @@ AUTHOR: Kreusada
 
 # Hosts
 
-Who are the hosts of the 2022 World Cup?:
+Which country were the hosts of the 2022 World Cup?:
```

Comment:
`were the hosts of` -> `was the host of`
`were the champions of` -> `was the champion of`
`were the runner ups of` -> `was the runner up of`
For all questions of these formats